tests/e2e/.env.example

```
@@ -0,0 +1,14 @@
+
+# Site under test.
+BASE_URL=http://localhost:8888
+
+# WP admin credentials — used for the UI login and the saved session.
+WP_ADMIN_USER=admin
+WP_ADMIN_PASSWORD=password
+
+# REST/API auth via a WP Application Password
+# (Users -> Profile -> Application Passwords). Spaces are part of the value.
+WP_API_USER=admin
+WP_APP_PASSWORD=xxxx xxxx xxxx xxxx xxxx xxxx
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)